### PR TITLE
FirstFlight-Megan Smith

### PR DESCRIPTION
**Pull request by mcs2515**

### CODE REVIEW

**Comment by ritjoe** [comment]
:+1:

File: people/2017/spring/mcs2515.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#hfoss
+name: Megan C. Smith
+avatar: http://en.gravatar.com/mcs2515
+blog: https://mcs2515.wordpress.com/
+feed: https://mcs2515.wordpress.com/feed/
+email: mcs2515@rit.edu
+major: Interactive New Media Development
+irc: mcs2515
+rit_dce: None
```
Comment (jflory7) [inline people/2017/spring/mcs2515.yaml]: @mcs2515 @ritjoe Just noticed this, but the `rit_dce` is blank here. This is just for your RIT computer account / username, e.g. mcs2515. You may want to send in another PR to add this to your profile, @mcs2515! 😃
